public/javascripts/Gallery/css/tags.css

```diff
@@ -12,6 +12,37 @@
     white-space: nowrap;
 }
 
+.gallery-tag {
+    width: 85%;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    display: inline-block;
+    padding: 0px 5px;
+    cursor: pointer;
+    border-style: solid;
+    border-width: 1px;
+    border-radius: 5px;
+    margin: 2px 2px;
+    text-align: left;
+    color: black;
+    white-space: nowrap;
+}
+
+.gallery-tag:hover {
```

Comment:
Similarly, for the hover, can you include only the properties that change?

Reply:
> This looks great @rpechuk ! Just wondering though, is it possible to make it so the shorter tags still have the border wrapping the text (kinda like that bubble look we had before). It saves a lot of sidebar real estate (especially if we plan on adding more things to the sidebar in the future). Thanks!

I'm not sure I understand what you mean do you mind explaining/drawing a quick sketch?

Also Mikey as soon as I get home I'll get that finished up👌🏻